content/nginx-one/rbac/roles.md

```diff
@@ -13,8 +13,8 @@ We provide three default **[roles](https://docs.cloud.f5.com/docs-v2/administrat
 
 ### Admin
 
-The Admin role, identified as `f5xc-nginx-one-admin`, provides full read and write access to all endpoints and features within the NGINX One Console. 
-It also supports RBAC for related XC services, as described in [Role-based Access Control Concepts](https://flatrender.tora.reviews/docs-v2/administration/how-tos/user-mgmt/rbac).
+The Admin role, identified as `f5xc-nginx-one-standard-admin`, provides full read and write access to all endpoints and features within the NGINX One Console. 
+It also supports Role-based access control for related XC services.<!--, as described in [Role-based Access Control Concepts](https://docs.cloud.f5.com/docs-v2/administration/how-tos/user-mgmt/rbac).-->
```

Comment:
I get a 404 error when I try to access this page.

![image](https://github.com/user-attachments/assets/69adbe3d-5242-4d86-8b6a-95898c320e81)